dts/x86/intel/raptor_lake.dtsi

```diff
@@ -161,47 +161,59 @@
 			compatible = "intel,penwell-spi";
 			vendor-id = <0x8086>;
 			device-id = <0x7aaa>;
-			#address-cells = <1>;
-			#size-cells = <0>;
 			pw,cs-mode = <0>;
 			pw,cs-output = <0>;
 			pw,fifo-depth = <64>;
 			cs-gpios = <&gpio_0_i 15 GPIO_ACTIVE_LOW>;
 			clock-frequency = <100000000>;
 			interrupts = <PCIE_IRQ_DETECT IRQ_TYPE_LOWEST_LEVEL_LOW 3>;
 			interrupt-parent = <&intc>;
+			spi_dma0: dma {
+				compatible = "intel,lpss";
```

Comment:
btw, just noticed it, but isn't that binding name a bit too short? dts/bindingt/dma/intel,lpss.yaml  should probably need no be moved to dts/bindings/dma/intel,lpss-dma.yaml (then the name is self-documenting). 
Looks like we, at intel, seem to name dma bindings weirdly compared to all others  :)  (adsp ones for instance)

Reply:
As I  said with I2C, this is not working with SPI devices. For example, when building with SPI loopback where we have overlay for RPL-S:

```
-- Found devicetree overlay:<PATH>/zephyrproject/zephyr/tests/drivers/spi/spi_loopback/boards/intel_rpl_s_crb.overlay
devicetree error: 'reg' property in <Node /pcie0/spi0/slow@0 in '<PATH>/zephyrproject/zephyr/misc/empty_file.c'> has length 4, which is not evenly divisible by 12 (= 4*(<#address-cells> (= 2) + <#size-cells> (= 1))). Note that #*-cells properties come either from the parent node or from the controller (in the case of 'interrupts').
```